### PR TITLE
Add generation of identifier for result values

### PR DESCRIPTION
**Pull request by tstrass**
Previously in the Procedure Performed template, we were calling `identifierForInterface` on the `Procedure` to generate identifiers for each `ResultValue`. This was bad because `identifierForInterface` will only work on `string`, `int64`, and `*int64`, and will otherwise always return the same string. 

This adds a new template helper function to generate an ID for a `ResultValue`. I also changed the Procedure Performed entry fixture to include a procedure that has result values, and added assertions to test that those are there in the xml.

### CODE REVIEW

File: exporter/cat1_view_helper.go [hunk 1]
```diff
@@ -83,6 +84,30 @@ func identifierForString(objs ...string) string {
 	return identifierFor([]byte(b))
 }
 
+// identifierForResultValue generates an MD5 representation of a ResultValue
+func identifierForResultValue(resultValue models.ResultValue) string {
+	var b []byte
+	codeSystemsInResultValue := make([]string, len(resultValue.Codes))
```
Comment (ljades) [inline exporter/cat1_view_helper.go]: Wouldn't you just initialize this the same way you do `codesInResultValue`? Because you only append later on, you don't need to run `make` with a starting length.

File: fixtures/entries/procedure_performed.json [hunk 2]
```diff
@@ -1,36 +1,62 @@
...
+  "time": null,
+  "values": [
+    {
       "codes": {
-        "ICD-10-PCS": [
-          "3E0F7DZ"
-        ],
         "SNOMED-CT": [
-          "24277005"
+          "369895002"
```
Comment (ljades) [inline fixtures/entries/procedure_performed.json:36]: What is the reason for the changes in this entry (not just on this line)? Is it updates from a newer bundle?

File: exporter/cat1_view_helper.go (hunk 1 above)
Comment (tstrass) [reply exporter/cat1_view_helper.go]: Agreed I'll just initialize it as a slice

File: fixtures/entries/procedure_performed.json (hunk 2 above)
Comment (tstrass) [reply fixtures/entries/procedure_performed.json:36]: No, this is an entirely different entry from the same bundle. So the codes are just different

Comment (tstrass) [reply fixtures/entries/procedure_performed.json:36]: Oh and the reason is in the description above- we needed a procedure performed that has result values so we can test that part of the template

**Review by mattrbianchi — APPROVED** [review]

**Review by dczulada — APPROVED** [review]
Looks good
